### DISCUSSION TITLE
HiSense A5CC and serial data

### DISCUSSION THREAD

**Discussion by BackSeat** [XCSoar/XCSoar#1477] [General]
I have data (GPS, FLARM and baro) as well as power being presented on a USB lead. When I plug that into my Pixel 5, XCSoar 7.42 can connect (IOIO UART 0) and sees the data. However, I'd like to use my HiSense A5CC phone. When I connect that, XCSoar reports "Device is not connected". I've checked baud rates and tried the other UART ports but no change.

I'd be grateful for any suggestions to help resolve this or point me in the right direction.

**Comment by lordfolken** [comment]
Do you see the UART ports in the device Ports list? If so the ioio has been detected and everything "should" work. 
Do you think this is a bug or a configuration issue?


If its a bug, a lot more information would be needed, 
- android os versions
- xcsoar logs from both devices
- configuration profiles from both devices
- logcat from both devices for the application
- potentially a device shipped to a developer.

**Comment by BackSeat** [comment]
Thanks for the quick response. 

Yes, the ports are listed in XCSoar.,

Bug or config? I don't know. I'm hoping there's some config option on the HiSense that I've not found yet.

I am expecting to be with the glider next weekend and I will capture the xcsoar logs. Not sure I can get logcat without an attached PC - is there a guide anywhere?

**Comment by BackSeat** [comment]
Sorry for the late update: I've been unwell. I found that USB debugging was enabled on the HiSense phone. When I turned that off, the data feed worked without problem.

Sorry for the noise and thanks for your help.
